content/articles/a-sbert-based-flask-web-app-for-automatic-text-summarization/index.md

```diff
@@ -0,0 +1,204 @@
+
+Copy the below html code in to the `summary.html` file:
+
+```html
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Your Summary</title>
+    <style>
+        p{
+            padding-top: 2em;
+            text-align: center;
+            line-height: 3em;           
+            color: black;
+            word-spacing: 0.25em;            
+            font-family: 'Times New Roman', Times, serif;
+        }
+    </style>
+    
+   <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3" crossorigin="anonymous">
+</head>
+<body>
+    <nav class="navbar navbar-light" style="background-color: hsl(236, 96%, 22%);">
+        <div class="container">
+            <a class="navbar-brand">Text SUMMARY</a>
+        </div>
+    </nav>
+    <p>{{ result }}</p>    
+</body>
+</html>
+```
+
+#### Step Three: Back-End coding
+In your project folder, create a file named `app.py` and should contain the below code.
+
+```python
+#importing flask
+from flask import Flask, render_template,request
+#Importing the summarizer
+from summarizer import Summarizer
+from summarizer.sbert import SBertSummarizer
+```
+
+The latest version of `bert-extractive-summarizer` lets you use `Sentence Bert`. It's based on [this](https://arxiv.org/abs/1908.10084) paper and [this]( https://www.sbert.net/) library.
+
+After importing libraries we create create a SBERT model. We'll use our SBERT model to summarize the required content in our next session.
+
+```python
+# Using an instance of SBERT to create the model
+model = SBertSummarizer('paraphrase-MiniLM-L6-v2')# Useful for assignments since it converts phrases and paragraphs into a 384-dimensional dense vector space.
+app = Flask(__name__)# If you want to know which URL should be used to call the associated method, use the route() function of Flask's class. The URL binding with the function is represented by the rule's rule argument.
+@app.route("/")#In this case, mapping the URLs to a function that will handle the logic for each individual URL
+def msg():
+    # Giving a result
+    return render_template('index.html')# displaying index.html contents
+@app.route("/summarize",methods=['POST','GET'])# If the gateway is unable to obtain the client's IP address, this information will be missing from the request.
+def getSummary():
+    body=request.form['data']# Sending a request
+    result = model(body, num_sentences=5)# Defining instances
+    return render_template('summary.html',result=result)# Displaying summary.html data
+if __name__ =="__main__":# When modules are imported, it allows or deny certain code to be executed.
+    app.run(debug=True,port=8000)#IP address on which a Web client can communicate with the server.
+```

```

Comment:
It's good that you've added comments. But, looks like the it's not very readable.

Hence, I suggest you to write the explanations below the code snippet as bulleted points using `-`.